tests/test_memory.py

```diff
@@ -0,0 +1,59 @@
+"""Python module that contains MemoryConnector unittests"""
+import pytest
+
+from happy_bank_core.data.memory import MemoryConnector
+from happy_bank_core.logic.account import Account
+
+
+class TestMemoryConnector:
+    """Class that contains MemoryConnector unittests"""
+
+    def setup_method(self):
+        """Setup method that creates list of account and MemoryConnector class instance"""
+        self.accounts = {
+            "id321": Account("id321", "Johan Doe", 1000),
+            "id123": Account("id123", "John Doe", 1000),
+            "id456": Account("id456", "Johanna Doe", 1000),
+        }
```

Comment:
move to setup_class method to create it only once when the test cases are running